modules/openapi-generator/src/main/resources/python/python-experimental/model_test.mustache

```diff
@@ -11,19 +11,19 @@ import unittest
 import {{packageName}}
 
 
-class Test{{unescapedDescription}}(unittest.TestCase):
```

Comment:
Please revert this, if we need to change this we should make this as a separate change to python-experimental
You can check the git blame of this line change, unescapedDescription is part or detailed edits done to models in python-experimental

Reply:
Done

Reply:
Are you sure that this has been done?
This comment also applies to lines 15, 23, 24, and 26 below
The PR still shows us using classname at those locations. Can you revert this file back to the master branch version?

Reply:
Done